### PR TITLE
[#279] Improve multiple based objects for tokenator values integration

### PR DESCRIPTION
**Pull request by pylapp**
_Note: Please transform `- [ ]` into `- (NA)` in the description when things are not applicable_

### Related issues

#279 

### Description

- Split multiple* objects
- Improve documentation
- Enrich tests

### Motivation & Context

Need to help integration of values generated by the tokenator, thus tokens it cannot manage must be moved elsewhere.

### Types of change

- Refactoring (non-breaking change)

#### Contribution

- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/ouds-ios/blob/develop/.github/CONTRIBUTING.md)

#### Accessibility

- (NA) My change follows accessibility good practices

#### Design

- [x] My change respects the design guidelines of _Orange Unified Design System_

#### Development

- [x] My change follows the [developer guide](https://github.com/Orange-OpenSource/ouds-ios/blob/develop/.github/DEVELOP.md)
- [x] I checked my changes do not add new SwiftLint warnings
- [x]  I have added unit tests to cover my changes _(optional)_

#### Documentation

- [x] My change introduces changes to the documentation and/or I have updated the documentation accordingly

### Checklist (for Core Team only)

- [x] The evolution have been tested and the project builds for iPhones and iPads
- [x] Code review has been done by reviewers according to [CODEOWNERS file](https://github.com/Orange-OpenSource/ouds-ios/blob/develop/.github/CODEOWNERS)
- (NA) Design review has been done
- (NA) Accessibiltiy review has been done
- (NA) Q/A team has tested the evolution
- [x] Documentation has been updated if relevant
- [x] Internal files have been updated if relevant (like CONTRIBUTING, DEVELOP, THIRD_PARTY, CONTRIBUTORS, NOTICE)
- [x] CHANGELOG has been updated respecting [keep a changelog rules](https://keepachangelog.com/en/1.0.0/) and reference the issues
- [x]  [The wiki](https://github.com/Orange-OpenSource/ouds-ios/wiki) has been updated if needed _(optional)_

### CODE REVIEW

**Comment by pylapp** [comment]
@ludovic35 Le message de commit peut être par exemple =

```text
refactor: improve management of tokens for those based in color scheme or size classes (#279) (#301)
```

File: OUDS/Core/OUDS/Sources/OUDSTheme/OUDSTheme+SemanticTokens/OUDSTheme+TypographyMultipleSemanticTokens.swift [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+// 
+// Authors: See CONTRIBUTORS.txt
+// Software description: A SwiftUI components library with code examples for Orange Unified Design System 
+//
+
+import Foundation
+import OUDSTokensRaw
+import OUDSTokensSemantic
+
+// [File not generated by the tokenator]
```
Comment (ludovic35) [inline OUDS/Core/OUDS/Sources/OUDSTheme/OUDSTheme+SemanticTokens/OUDSTheme+TypographyMultipleSemanticTokens.swift:18]: is it right ? gnerated ?

**Review by Tayebsed93 — DISMISSED** [review]

File: OUDS/Core/OUDS/Sources/OUDSTheme/OUDSTheme+SemanticTokens/OUDSTheme+SpaceMultipleSemanticTokens.swift [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+// WARNING: Not synchronized anymore with the Figjam / Figma by developers team
+// Create an issue for update https://github.com/Orange-OpenSource/ouds-ios/issues/new?template=token_update.yml
+
+/// Defines wrapper objects for space semantic tokens (i.e. in the end ``DimensionRawToken`).
+/// These values can be overriden inside ``OUDSTheme`` subclasses (in extensions or not, in the same module or not) thanks to the `@objc open` combination.
+extension OUDSTheme: SpaceMultipleSemanticTokens {
+
+    // MARK: Semantic token - Spacing - Layout fluid
+
+    @objc open var spaceScaledNone: MultipleSpaceTokens { MultipleSpaceTokens(compact: spaceScaledNoneMobile, regular: spaceScaledNoneTablet) }
```
Comment (ludovic35) [inline OUDS/Core/OUDS/Sources/OUDSTheme/OUDSTheme+SemanticTokens/OUDSTheme+SpaceMultipleSemanticTokens.swift:28]: For all Multiple token should we prefer MultipleSementic ?
Example MultipleSpaceTokens shoould be something like SpaceMultipleSementicToken

**Review by ludovic35 — CHANGES_REQUESTED** [review]

File: OUDS/Core/OUDS/Sources/OUDSTheme/OUDSTheme+SemanticTokens/OUDSTheme+TypographyMultipleSemanticTokens.swift (hunk 1 above)
Comment (pylapp) [reply OUDS/Core/OUDS/Sources/OUDSTheme/OUDSTheme+SemanticTokens/OUDSTheme+TypographyMultipleSemanticTokens.swift:18]: Ce fichier n'est pas généré par le _tokenator_ car il ne peut pas générer de tels objets dépendant de _size class_ ou _color scheme_ dans le mesure où quand bien même ces tokens sont définis sur Figma par l'équipe de design cet outil n'est _a priori_ pas capable de les gérer. De fait le JSON produit pas Figma ne peut pas ressortir ces objets, donc le _tokenator_ ne peut rien produire. D'où _in fine_ le fait qu el'on expose ces objets en pointant nous-mêmes vers les tokens pertinents.

**Review by Tayebsed93 — APPROVED** [review]

**Pull request merged**